plugins/cairo-lang-macro/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cairo-lang-macro"
-version = "0.1.1"
+version = "0.2.0"
```

Comment:
just an idea: can't we 1.0 it ?

Reply:
Possibly, but probably could be discussed more broadly.

Reply:
The argument would be that we should have the full implementation of initially planned scope (we know already that we would finally need to have text spans when we have first implemented macros) 🤔

Reply:
Anyway, doesn't need to be decided here, added to #2102